### PR TITLE
zed: yoga merge

### PR DESCRIPTION
**Pull request by markgoddard**
- Fix query for top Ceph pools by capacity used
- Default to RegionOne for os-capacity
- Moving variable from play to group_vars This lets us override in case one of the machines have different VG name Otherwise play variables take priority.
- Document update of IPA kernel URL
- Add docs page for running Tempest
- Bump magnum-capi-helm version to latest
- Bump Magnum
- bump magnum tag
- reno
- Fix os-capacity playbook crash on delegate_to
- Improved AIO deployment script
- magnum_tag
- docs: Update microversions in tempest.conf example for Zed
- Update magnum container images for Zed

### CODE REVIEW

File: doc/source/operations/tempest.rst [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+    # Add the repository to Apt sources:
+    echo \
+    "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+    $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+    sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+Installing Docker on CentOS/Rocky:
```
Comment (Alex-Welsh) [inline doc/source/operations/tempest.rst:68]: nit: no more centos
```suggestion
Installing Docker on Rocky:
```

File: doc/source/operations/tempest.rst [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+
+Build a Kayobe automation image:
+
+.. code-block:: bash
+
+    git submodule init
+    git submodule update
+    # If running on Ubuntu, the fact cache can confuse Kayobe in the CentOS-based container
+    mv etc/kayobe/facts{,-old}
+    sudo DOCKER_BUILDKIT=1 docker build --file .automation/docker/kayobe/Dockerfile --tag kayobe:latest .
```
Comment (Alex-Welsh) [inline doc/source/operations/tempest.rst:104]: Currently still need the extra build arg for a rocky 9 container
```suggestion
.. code-block:: bash

    git submodule init
    git submodule update
    # If running on Ubuntu, the fact cache can confuse Kayobe in the Rocky linux based container
    mv etc/kayobe/facts{,-old}
    sudo DOCKER_BUILDKIT=1 docker build --build-arg BASE_IMAGE=rockylinux:9 --file .automation/docker/kayobe/Dockerfile --tag kayobe:latest .
```

**Review by Alex-Welsh — CHANGES_REQUESTED** [review]

File: doc/source/operations/tempest.rst (hunk 1 above)
Comment (markgoddard) [reply doc/source/operations/tempest.rst:68]: If you don't mind I'll follow up with these changes because we have a successful CI run.

**Comment by markgoddard** [comment]
https://github.com/stackhpc/stackhpc-kayobe-config/pull/970

**Pull request merged**